torch_geometric/data/temporal.py

```diff
@@ -207,6 +268,16 @@ def num_events(self) -> int:
         """
         return self.src.size(0)
 
+    @property
+    def num_snapshots(self) -> int:
```

Comment:
do you think we should just expose `unique_times`? That way people also can see gaps between the snapshots?